composer.lock

```diff
@@ -1209,7 +1209,7 @@
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/guzzle/guzzle/zipball/9c43893a8e580d8401e4210cebe038d850341012",
+                "url": "https://api.github.com/repos/guzzle/guzzle/zipball/66737d2024d49c06bd294f33d60eb369f18b20f2",
```

Comment:
I think this is something that guzzle did one their end. When I am adding this dependency or not, it still updates that tarball url for guzzle. I think they changed the url on their end for that same reference number (notice how the reference hash for that block is the same as before).